### ISSUE TITLE
HIChartView.addFont(path) does not work after updating to new version of SDK 8.0.4

### ISSUE THREAD

**Issue by gasta-java** [highcharts/highcharts-ios#306]
Hi!

HIChartView.addFont(path) does not work after updating to new version of SDK 8.0.4.
But it works okay in version 8.0.0

1) I add: 
```
let path = Bundle.main.path(forResource: "Barlow-Regular", ofType: "ttf")
HIChartView.addFont(path)
```
2) Then I use in many places, but it does not work anymore and does not apply anywhere; 

Testing env:
SWIFT, iOS 13.2.2, HC SDK 8.0.4

**Comment by ihnatmoisieiev** [comment]
Hello @gasta-java,

thank you for reporting the issue. It will be fixed in the next release.

**Comment by mansoor92** [comment]
Can you please tell me when next release is expected.
Is there a workaround or prerelease that i can use.
Thanks

**Comment by ihnatmoisieiev** [comment]
@gasta-java the issue has been fixed in the latest 8.1.0 version. Can I close it?
